assets/js/googlesitekit/datastore/user/date-range.test.js

```diff
@@ -21,8 +21,10 @@
  */
 import { createTestRegistry } from '../../../../../tests/js/utils';
 import { STORE_NAME } from './constants';
+import { getPreviousDate, getDateString, getPreviousWeekDate } from './utils';
 
 describe( 'core/user date-range', () => {
+	const today = getDateString( new Date() );
```

Comment:
Looking at the tests here, most of them will need to be updated due to the return value change required above. However, one other concern: The tests themselves rely on all the utility functions which are already tested elsewhere, and I feel like it would be more appropriate to test specific hard-coded input dates and result dates, relying on only `getDateRangeDates` calls instead.

How about the following idea:
* We add another optional `options.referenceDate` to `getDateRangeDates`, which if not provided will be set to "today" (i.e. what you have above).
* This argument wouldn't be used in production code, but is helpful for testing.
* This way all the tests could be updated to work with hard-coded dates so that they no longer rely on other utility functions, ensuring that the actual selector works as expected, regardless of the utility functions (and whether they may unexpectedly change).

Reply:
`options.referenceDate` has been added & tests have been modified to use static dates.